apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Tests/BigtableClientTest.cs

```diff
@@ -195,6 +195,50 @@ private class TestBigtableClient : BigtableClient { }
                 () => client.MutateRowAsync(tableName, rowKey, mutations, CallSettings.FromCancellationToken(default)));
         }
 
+        [Fact]
+        public void MutateRows_Valid_Request()
+        {
+            var client = new TestBigtableClient();
+            var tableName = new TableName("project", "instance", "table");
+
+            // NotImplementedException means it got through the normal validations and tried to actually
```

Comment:
Not for this PR, but maybe it would be useful to have something that through a test-only "MadeItToServiceException" to avoid this sort of comment being worthwhile. (I think we'd want that in generic test infrastructure, not Bigtable-specific.)

Reply:
Good idea. It's not too much churn, so I'll do it specific to this for now, but yes perhaps something common would be good in the future.